receiver/nvmlreceiver/testcudakernel/test_cuda_kernel.cc

```diff
@@ -16,30 +16,33 @@
 // https://stackoverflow.com/questions/27900849/unable-to-use-cublasxt
```

Comment:
Oops, I forgot I had my editor configured to run `clang-format` on save for C/C++ files. Shall I revert the formatting changes?

Reply:
I think we can keep this - no need to revert.